### DISCUSSION TITLE
How to exclude questions about flags?

### DISCUSSION THREAD

**Discussion by Johan795** [axelboc/anki-ultimate-geography#653] [Q&A]
Hello,

sorry if this question is basic, but how do I do the get questions about everything except the flags?

(or maybee to order all the questions about flags to show up  after everything else)

**Comment by aplaice** [comment]
Hi,

No problem — one option would be to create a filtered deck ([see here](https://github.com/anki-geo/ultimate-geography/blob/master/README.md#custom-study)). The search query that you'd need would be `deck:"Ultimate Geography" -card:"Flag - Country" -card:"Country - Flag"`.

Alternatively, you could just suspend the flag-related cards — go to the Anki "browser" (<kbd>b</kbd>), search for `card:"Flag - Country" card:"Country - Flag"`, select all (<kbd>Ctrl</kbd>+<kbd>a</kbd>) and suspend (<kbd>Ctrl</kbd>+<kbd>j</kbd>).

Hope this helps!

**Reply by Johan795** [reply]
Great, thank You very much for the answer!
